README.md

```diff
@@ -112,6 +112,24 @@ Before building we need to set up the golang environment. Do this by running:
 make install
 ```
 
+Ensure, that environment is set correctly:
+
+```bash
+make go-env-test
+```
+
+**CGO_CFLAGS** must be set to "-I<absolute_path_to_repo>/go-spacemesh/build/ -DSQLITE_ENABLE_DBSTAT_VTAB=1"
+**CGO_LDFLAGS** must be set to "-L<absolute_path_to_repo>/go-spacemesh/build/ -Wl,-rpath,$ORIGIN -Wl,-rpath,<absolute_path_to_repo>/go-spacemesh/build/"
+
+Ensure you have installed **OpenCL** library
```

Comment:
```suggestion
Make sure you have **OpenCL** library installed
```
but also we probably want to have version specified

Reply:
Both "make sure" and "ensure" are correct.